packages/nc-gui/pages/signin.vue

```diff
@@ -121,6 +121,19 @@ function resetError() {
             </div>
           </div>
         </a-form>
+        <div>
+          <a-button
+            v-if="appInfo.googleAuthEnabled"
+            :href="`${appInfo.ncSiteUrl}/auth/google`"
+            outlined
+            large
+            elevation-10
+            block
+            color="blue"
+          >
```

Comment:
For styling reasons could you use this instead?

```
<a
  v-if="appInfo.googleAuthEnabled"
  :href="`${appInfo.ncSiteUrl}/auth/google`"
  class="scaling-btn bg-opacity-100 after:(!bg-white) !text-primary !no-underline"
>
  <span class="flex items-center gap-2">
    <LogosGoogleGmail />

    Sign In with Google
  </span>
</a>
```

Reply:
<img width="521" alt="Screenshot 2022-10-05 at 15 08 08" src="https://user-images.githubusercontent.com/17380265/194068061-e7a0ca5a-5204-4543-bb28-1f2c8937702d.png">
It looks like this

Reply:
I opened a new PR as suggested :) 
Thank you for your contribution!